.github/pull_request_template.md

```diff
@@ -1,3 +1,5 @@
+> 🚨 Note: If you are not a project maintainer (i.e. you do not work at Slack), please use [this issue form](https://forms.gle/jHS17eYEYAtcXA96A), or for feedback, please use [this feedback form](https://forms.gle/B2PRF9HQheRgQdo7A). 🚨
+
 (To the project maintainers, you can remove the following and write the description of this pull request)
```

Comment:
We may want to move this line to the top. Also, the two "Note" lines can be merged into one. Thoughts?

Reply:
Yes, good point. I will do that now.